src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
Even something more realistic like
```
me.sys.updatesPerSecond = 60;
me.sys.fps = 30;
```
Let's say you're doing this for more accurate collision checks at 60fps. If so, why limit your drawing to 30fps? Why not let the browser draw at 60fps if it can? I'm just not seeing the use case here.